oclubs/static-dev/js/newreservation.js

```diff
@@ -0,0 +1,76 @@
+(function($) {
+  $(document)
+    .ready(function() {
+
+      $(document).ajaxStart(function() {
+        $('.loading_gif').show();
+        $('.finished_loading').hide();
+      });
+
+      $(document).ajaxStop(function() {
+        $('.loading_gif').hide();
+        $('.finished_loading').show();
+
+
+				Date.prototype.today = function() {
```

Comment:
Inconsistent indentation

Reply:
Also, having your own methods is preferred then writing to a built-in's prototype